src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -737,6 +741,13 @@ private function getSchemeAndHierarchy(string $filename): array
         return 2 === \count($components) ? [$components[0], $components[1]] : [null, $components[0]];
     }
 
+    private static function assertOperationIsAvailable(string $functionName): void
```

Comment:
Why static function is implemented here? Just wondering if We can use non-static function?

Reply:
We cannot anymore since this is now called from an existing static function.
For private methods, static can be a bit faster and has no downside from a design pov.